Control/public/Model.js

```diff
@@ -190,9 +190,11 @@ export default class Model extends Observable {
     switch (this.router.params.page) {
       case 'environments':
         this.environment.getEnvironments();
+        this.environment.getEnvironmentRequests();
         this.frameworkInfo.getIntegratedServicesInfo();
         this.environment.refreshInterval = setInterval(() => {
           this.environment.getEnvironments();
+          this.environment.getEnvironmentRequests();
```

Comment:
Currently the production setup is set to 2 minutes and 1 second for a refresh. 
For now, I think we would need a different timer for pending requests refresh.
We cannot reduce the existing ones as "getEnvironments" would spam AliECS with too many requests.

Reply:
This will be addressed by get envs cache.